code/about.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en"
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wer steckt hinter Teigträume Schachner?</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+    <a href="./">Willkommen!</a>
+<section>
+    <h1> Wer steckt hinter Teigträume Schachner?
+    </h1>
+    <aside> <img class="Foto-vertikal" src="../img/Melanie1.jpg">
+        <div>
+        <p>Ich bin Melanie. Betriebswirtin (FH) und Inhaberin eines französischen Bäckerdiploms ("CAP Artisan boulanger bio").
+        </p>
+        <p>Besondere Konstellation wirst du dir denken...
+        </p>
+        </div>    
+    </aside>
+</section>
+<section>
+    <h2> Wie das kam? 
+    </h2>
+    <p>Durch meine persönliche Freude an gutem Brot und ein besonderes Erlebnis:
+    </p>
+    <p> Im Alter von 16 Jahren war ich im Rahmen eines Schüleraustauschs zum ersten Mal in Frankreich. Meine Gastfamilie hat keine Gelegenheit ausgelassen, mir die Besonderheiten der lokalen Küche vorzuführen. Eine hat sich mir besonders eingeprägt: das Baguette.

```

Comment:
Here you forgot a closing `</p>` tag 🙃
I think it would be easier to read your code if you put e.g. the h2 opening and closing tags on the same line as your headline. Same with the p tag.